framework/helpers/BaseConsole.php

```diff
@@ -1051,4 +1052,54 @@ public static function endProgress($remove = false, $keepPrefix = true)
         self::$_progressEtaLastDone = 0;
         self::$_progressEtaLastUpdate = null;
     }
+
+    /**
+     * Generates a summary of the validation errors.
+     * @param Model|Model[] $models the model(s) whose validation errors are to be displayed.
+     * @param array $options the tag options in terms of name-value pairs. The following options are specially handled:
+     *
+     * - showAllErrors: boolean, if set to true every error message for each attribute will be shown otherwise
+     *   only the first error message for each attribute will be shown. Defaults to `false`.
+     *
+     * @return string the generated error summary
+     * @since 2.0.14
+     */
+    public static function errorSummary($models, $options = [])
+    {
+        $showAllErrors = ArrayHelper::remove($options, 'showAllErrors', false);
+        $lines = self::collectErrors($models, $showAllErrors);
+
+        return implode("\n", $lines);
```

Comment:
Isn't it better to use `PHP_EOL` instead of `"\n"`?

Reply:
Yes, you are right 👍